src/lib/icons/file_dicom.svg

```diff
@@ -1 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="160" viewBox="0 0 32 32"><path fill="#fff" d="M25 27H7V5h13l5 5v17"/><path fill="#d0011b" d="M20 4H7a1 1 0 0 0-1 1v22a1 1 0 0 0 1 1h18a1 1 0 0 0 1-1V9.89zm5 23H7V5h13v4a1 1 0 0 0 1 1h4z"/><path fill="#d0011b" d="M10 19h11v1H10zM10 21h11v1H10zM15 23h6v1h-6zM16.55 12.44V10h-2.1V12.44H12v2.12h2.45V17h2.1v-2.44H19v-2.12h-2.45"/></svg>
```

Comment:
We're missing a DICOM icon in the new set. Will update once I hear back from the design team.